src/server/pkg/dbutil/dbutil.go

```diff
@@ -0,0 +1,65 @@
+package dbutil
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/jinzhu/gorm"
+	"github.com/jmoiron/sqlx"
+	"github.com/pachyderm/pachyderm/src/client/pkg/require"
+)
+
+// set this to true if you want to keep the database around
+var devDontDropDatabase = false
+
+const (
+	DefaultPostgresHost = "127.0.0.1"
+	DefaultPostgresPort = 32228
+	TestPostgresUser    = "postgres"
+)
+
+// WithTestDB connects to postgres using the default settings, creates a database with a unique name
+// then calls cb with a sqlx.DB configured to use the newly created database.
+// After cb returns the database is dropped.
+func WithTestDB(t testing.TB, cb func(db *sqlx.DB)) {
+	dsn := fmt.Sprintf("host=%s port=%d user=%s sslmode=disable", DefaultPostgresHost, DefaultPostgresPort, TestPostgresUser)
+	db := sqlx.MustOpen("postgres", dsn)
+	dbName := fmt.Sprintf("test_%d", time.Now().UnixNano())
+	db.MustExec("CREATE DATABASE " + dbName)
+	t.Log("database", dbName, "successfully created")
```

Comment:
Was this just for debugging?

Reply:
It's so you have access to the database name in the test.

Reply:
It is for debugging, but I think it's good to have in the test logs all the time.